dev/io.openliberty.io.smallrye.reactive.messaging-provider4/bnd.bnd

```diff
@@ -59,7 +62,9 @@ Private-Package: \
   com.ibm.ws.cdi.interfaces.jakarta;version=latest,\
   com.ibm.wsspi.org.osgi.service.component.annotations;version=latest,\
   com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
+  com.ibm.ws.container.service;version=latest,\
```

Comment:
Not sure why you need this dependency, I don't see any reference to `com.ibm.ws.container.service` packages in this bundle.

Reply:
`com.ibm.ws.threadContext.ComponentMetaDataAccessorImpl` is used to obtain the application name currently to that we know which application if multiple are installed.

And that package is part of that bundle, so it is used. if there is a better way to get the app name then we can replace it